delete-netlify/index.js

```diff
@@ -0,0 +1,74 @@
+const axios = require("axios");
```

Comment:
can remove sites.txt

Reply:
prefer not to - that's the file which we read from

Reply:
okay